src/consensus/pbft/pbft.h

```diff
@@ -244,12 +269,21 @@ namespace pbft
     // https://github.com/microsoft/CCF/issues/57
     View get_view() override
     {
-      return 2;
+      return message_receiver_base->view() + 2;
```

Comment:
why +2?

Reply:
This is a hold over from Raft terms. I will fix this later.